templates/al2023/provisioners/hardened-images.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+if [[ "$HARDENED_IMAGE" == "true" ]]; then
```

Comment:
what are the downsides of setting these by default? where would this pose an issue?

Reply:
Sorry my backlogs been rather large recently and missed this. Providing the binary exists by default then I can't foresee any.